Client/src/Components/DataCell/DataCell.js

```diff
@@ -42,7 +44,7 @@ export default function DataCell() {
             await fetch('http://localhost:4000/dataupload', options)
                 .then(resp => resp.json())
                 .then(result => {
-                    console.log(result[0])
+                    console.log(result)

```

Comment:
No need to log here

Reply:
I asked him to change this because the logged message was incomplete. In the console browser, after uploading a dataset, it would log "U" only instead of "Upload was successful!"

Reply:
I don't think its a good idea to remove the console logs at the moment since they are mostly for troubleshooting the file upload and im assuming other people are working with it as well. 
If required, i'll remove them on a future commit.